src/Components/InstanceTypes/SuggestedInstanceTypesTable.js

```diff
@@ -71,6 +79,21 @@ export default function SuggestedInstanceTypesTable() {
                                 onSetPage,
                                 onPerPageSelect
                             }}
+                            filterConfig={{
+                                items: [{
+                                    label: 'instance types',
+                                    type: 'text',
+                                    filterValues: {
+                                        onChange: (e, value) => setInstanceTypeName(value),
+                                        value: instanceTypeName
+                                    }
+                                }]
+                            }}
+                            activeFiltersConfig={{
+                                deleteTitle: 'Reset filters',
+                                filters: activeFilters(),
+                                onDelete: () => setInstanceTypeName('')
```

Comment:
@r14chandra I am curious to know how this will work when we have multiple chips and we try to delete them one by one (using the cross near chip)? have you tried it?

Reply:
I guess, we need to add another callback function. I have not tried though.